envs/secrets.env.template

```diff
@@ -6,3 +6,5 @@ API_MAILJET_SECRET=set_it
 
 API_EMPLOI_STORE_KEY=set_it
 API_EMPLOI_STORE_SECRET=set_it
+
+PDFSHIFT_API_KEY=set_it
```

Comment:
new line needed (didn't `make pylint` detect it?)

Reply:
Non, Pylint n'a rien détecté du tout ! :cry: